### ISSUE TITLE
No response data in Error Case

### ISSUE THREAD

**Issue by loiu92** [developit/redaxios#48]
Hello,

I just try your module and it's working well.
I just don't have any response object when HTTP Code is 422

```
body: ReadableStream { locked: false }
bodyUsed: false
config: Object { headers: {…} }
headers: Headers {  }
ok: false
redirected: false
status: 422
statusText: "No Reason Phrase"
type: "cors"
url: "https://url"
<prototype>: {…}
```

But in Axios, i have in the returned object, a `response` field then a `data` with my json.

Did i do something wrong (i didn't change anything from my axios implementation)

Thanks in advance :)

**Comment by auburnsummer** [comment]
It looks like at line 204-205 it checks for a 4xx/5xx status before parsing the response data, so there won't be response data on error responses. 

For me, I just copied the entire index.js into my own project. Then I removed line 205 (`if  (!ok)...`) and changed line 219 to `.then(() => ok ? response : Promise.reject(response))`. Seemed to work fine.

**Comment by developit** [comment]
Fixed in #56

**Comment by mixalturek** [comment]
Hi, any plan for release of the fix?

**Comment by orinayo** [comment]
@developit, do you plan on releasing this fix soon?

**Comment by andreychev** [comment]
@developit 

Any news?

**Comment by developit** [comment]
@andreychev I just published 0.4.1 with this included.

**Comment by axmad386** [comment]
Hi @developit. I have updated redaxios. Before it, I use axios and this is my error handler example
```js
Axios(...)
 .then(..)
 .catch(err=>{
    if(err.response!== undefined){
    // handle error data here
    console.log(err.response.data)
 })
```
But in redaxios@0.4.1 i have to refactor the code to this
```js
Axios(...)
 .then(..)
 .catch(err=>{
    if(err.data!== undefined){
    // handle error data here
    console.log(err.data)
 })
```
Note the difference, `err.response.data (axios)` vs `err.data(redaxios)`
